docs/generate_docs.go

```diff
@@ -118,7 +118,69 @@ func generateServerOptions(system *core.System) {
 		}
 	})
 
-	generatePartitionedConfigOptionsDocs("Server Options", "docs/pages/deployment/server_options.rst", flags)
+	// We want 2 tables, one with all did:web-related and relevant core flags,
+	// the rest (which is v5/did:nuts/gRPC-specific), goes into a second table below it.
+	// The predicates below define which are v5-related and which are not.
+	v5FlagsPredicates := []func(f *pflag.Flag) bool{
+		func(f *pflag.Flag) bool {
+			return f.Name == "internalratelimiter"
+		},
```

Comment:
the `internalratelimiter` also applies to did:web VC issuance, so maybe keep this in the standard lis for now?